h2o-genmodel/src/main/java/hex/genmodel/easy/EnumEncoderColumnMapper.java

```diff
@@ -7,14 +7,19 @@
 
 public class EnumEncoderColumnMapper {
 
-  private final GenModel _m;
+  final GenModel _m;
+  String[] modelColumnNames;
```

Comment:
there is no reason to make this a instance variable a simple protected getModelColumnNames() method would be enough, together with making _m protected

Reply:
oh i see, fixed

Reply:
👍